monkestation/code/modules/mob/living/carbon/emote.dm

```diff
@@ -0,0 +1,80 @@
+//Finger Guns *blows smoke off his finger tips*
+/datum/emote/living/carbon/fingergun
+	key = "fingergun"
+	key_third_person = "fingerguns"
+	hands_use_check = TRUE
+	cooldown = 1 SECONDS
+
+/datum/emote/living/carbon/fingergun/run_emote(mob/user, params, type_override, intentional)
+	. = ..()
+	if(!.)
+		return
+	var/obj/item/gun/ballistic/fingergun_emote/N = new(user)
+	if(user.put_in_hands(N))
+		to_chat(user, span_notice("You fold your hand into a finger gun."))
+		user.manual_emote("folds thier hand into a finger gun.")
```

Comment:
```suggestion
		user.manual_emote("folds [user.p_their()] hand into a finger gun.")
```

Reply:
oh I already added this but never pushed the commit lol